dizoo/atari/envs/atari_muzero_env.py

```diff
@@ -0,0 +1,202 @@
+        self.current_player_index = self.agents.index(current_agent)
+        self.agent_str = self._agent_selector.next()  # next player to play
+        action = int(action)
+        obs, reward, done, info = self._env.step(action)
+        # self._env.render()
+        self._final_eval_reward += reward
+        self.obs = to_ndarray(obs)
+        self.reward = to_ndarray([reward])
+        observation = self.observe()
+        info = {'next player to play': self.agent_str}
+
+        if done:
+            info['final_eval_reward'] = self._final_eval_reward
+
+        return BaseEnvTimestep(observation, self.reward, done, info)
+
+    def legal_actions(self):
+        return np.arange(self._action_space.n)
+
+    @property
+    def observation_space(self) -> gym.spaces.Space:
+        return self._observation_space
+
+    @property
+    def action_space(self) -> gym.spaces.Space:
+        return self._action_space
+
+    @property
+    def reward_space(self) -> gym.spaces.Space:
+        return self._reward_space
+
+    def random_action(self):
+        action_list = self.legal_actions()
+        return np.random.choice(action_list)
+
+    def render(self, mode='human'):
+        self._env.render()
+
+    def human_to_action(self):
+        """
+        For multiplayer games, ask the user for a legal action
+        and return the corresponding action number.
+        Returns:
+            An integer from the action space.
+        """
+        while True:
+            try:
+                print(f"Current available actions for the player {self.to_play()} are:{self.legal_actions()}")
+                choice = int(
+                    input(
+                        f"Enter the index of next move for the player {self.to_play()}: "
+                    )
+                )
+                if choice in self.legal_actions():
+                    break
+            except KeyboardInterrupt:
+                sys.exit(0)
+            except Exception as e:
+                print("Wrong input, try again")
+        return choice
+
+    def action_to_string(self, action_number):
+        """
+        Convert an action number to a string representing the action.
+        Args:
+            action_number: an integer from the action space.
+        Returns:
+            String representing the action.
+        """
+        return str(action_number)
+
+    def close(self) -> None:
+        if self._init_flag:
+            self._env.close()
+        self._init_flag = False
+
+    def seed(self, seed: int, dynamic_seed: bool = True) -> None:
+        self._seed = seed
+        self._dynamic_seed = dynamic_seed
+        np.random.seed(self._seed)
+
+    def __repr__(self) -> str:
+        return "DI-engine Atari Env({})".format(self.cfg.env_name)
+
+    @staticmethod
+    def create_collector_envcfg(cfg: dict) -> List[dict]:
+        collector_env_num = cfg.pop('collector_env_num')
+        cfg = copy.deepcopy(cfg)
+        cfg.is_train = True
```

Comment:
atari-game env doesn't use `is_train`, remove it, and what is the differences between collect and eval phase for atari-game env

Reply:
thanks, it has been modified